src/ApplicationConfig.php

```diff
@@ -73,14 +78,21 @@ public function offsetExists($offset)
     }
 
     /**
+     * @param mixed $offset
```

Comment:
The reason I didn't add these is because the types are defined in the interface - that said, I'm not sure how I feel about this when the interface is an internal PHP one. Thoughts?